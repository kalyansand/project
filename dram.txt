2 0 ACT0 0 0 0x0
2 0 ACT1 0 0 0x0
80 0 RD0  0 0 0x00
80 0 RD1  0 0 0x00
176 0 PRE  0 0 

178 0 ACT0 1 1 0x3f
178 0 ACT1 1 1 0x3f
256 0 WR0  1 1 0x1f
256 0 WR1  1 1 0x1f
352 0 PRE  1 1 

354 0 ACT0 2 2 0x1
354 0 ACT1 2 2 0x1
432 0 RD0  2 2 0x2e
432 0 RD1  2 2 0x2e
528 0 PRE  2 2 

530 0 ACT0 3 3 0x2
530 0 ACT1 3 3 0x2
530 0 ACT1 3 3 0x2
608 0 RD0  3 3 0x3d
608 0 RD1  3 3 0x3d
704 0 PRE  3 3 

706 0 ACT0 4 0 0x4
706 0 ACT1 4 0 0x4
784 0 WR0  4 0 0x4c
784 0 WR1  4 0 0x4c
880 0 PRE  4 0 

882 0 ACT0 5 1 0x8
882 0 ACT1 5 1 0x8
960 0 RD0  5 1 0x5b
960 0 RD1  5 1 0x5b
1056 0 PRE  5 1 

1058 0 ACT0 6 2 0x10
1058 0 ACT1 6 2 0x10
1136 0 RD0  6 2 0x6a
1136 0 RD1  6 2 0x6a
1232 0 PRE  6 2 

1234 0 ACT0 7 3 0x20
1234 0 ACT1 7 3 0x20
1312 0 WR0  7 3 0x79
1312 0 WR1  7 3 0x79
1408 0 PRE  7 3 

1410 0 ACT0 0 0 0x10
1410 0 ACT1 0 0 0x10
1488 0 RD0  0 0 0x88
1488 0 RD1  0 0 0x88
1584 0 PRE  0 0 

1586 0 ACT0 1 1 0x8
1586 0 ACT1 1 1 0x8
1664 0 RD0  1 1 0x97
1664 0 RD1  1 1 0x97
1760 0 PRE  1 1 

1762 0 ACT0 2 2 0x4
1762 0 ACT1 2 2 0x4
1840 0 WR0  2 2 0xa6
1840 0 WR1  2 2 0xa6
1936 0 PRE  2 2 

1938 0 ACT0 3 3 0x2
1938 0 ACT1 3 3 0x2
2016 0 RD0  3 3 0xb5
2016 0 RD1  3 3 0xb5
2112 0 PRE  3 3 

2114 0 ACT0 4 0 0x1
2114 0 ACT1 4 0 0x1
2192 0 RD0  4 0 0xc4
2192 0 RD1  4 0 0xc4
2288 0 PRE  4 0 

2290 0 ACT0 5 1 0x0
2290 0 ACT1 5 1 0x0
2368 0 WR0  5 1 0xd3
2368 0 WR1  5 1 0xd3
2464 0 PRE  5 1 

2466 0 ACT0 6 2 0x1
2466 0 ACT1 6 2 0x1
2544 0 RD0  6 2 0xe2
2544 0 RD1  6 2 0xe2
2640 0 PRE  6 2 

2642 0 ACT0 7 3 0x2
2642 0 ACT1 7 3 0x2
2720 0 RD0  7 3 0xf1
2720 0 RD1  7 3 0xf1
2816 0 PRE  7 3 

2818 0 ACT0 0 0 0x4
2818 0 ACT1 0 0 0x4
2896 0 WR0  0 0 0x100
2896 0 WR1  0 0 0x100
2992 0 PRE  0 0 

2994 0 ACT0 1 1 0x8
2994 0 ACT1 1 1 0x8
3072 0 RD0  1 1 0x11f
3072 0 RD1  1 1 0x11f
3168 0 PRE  1 1 

3170 0 ACT0 2 2 0x10
3170 0 ACT1 2 2 0x10
3248 0 RD0  2 2 0x12e
3248 0 RD1  2 2 0x12e
3344 0 PRE  2 2 

3346 0 ACT0 3 3 0x20
3346 0 ACT1 3 3 0x20
3424 0 WR0  3 3 0x13d
3424 0 WR1  3 3 0x13d
3520 0 PRE  3 3 

3522 0 ACT0 4 0 0x10
3522 0 ACT1 4 0 0x10
3600 0 RD0  4 0 0x14c
3600 0 RD1  4 0 0x14c
3696 0 PRE  4 0 

3698 0 ACT0 5 1 0x8
3698 0 ACT1 5 1 0x8
3776 0 RD0  5 1 0x15b
3776 0 RD1  5 1 0x15b
3872 0 PRE  5 1 

3874 0 ACT0 6 2 0x4
3874 0 ACT1 6 2 0x4
3952 0 WR0  6 2 0x16a
3952 0 WR1  6 2 0x16a
4048 0 PRE  6 2 

4050 0 ACT0 7 3 0x2
4050 0 ACT1 7 3 0x2
4128 0 RD0  7 3 0x179
4128 0 RD1  7 3 0x179
4224 0 PRE  7 3 

4226 0 ACT0 0 0 0x1
4226 0 ACT1 0 0 0x1
4304 0 RD0  0 0 0x188
4304 0 RD1  0 0 0x188
4400 0 PRE  0 0 

4402 0 ACT0 1 1 0x0
4402 0 ACT1 1 1 0x0
4480 0 WR0  1 1 0x197
4480 0 WR1  1 1 0x197
4576 0 PRE  1 1 

4578 0 ACT0 2 2 0x1
4578 0 ACT1 2 2 0x1
4656 0 RD0  2 2 0x1a6
4656 0 RD1  2 2 0x1a6
4752 0 PRE  2 2 

4754 0 ACT0 3 3 0x2
4754 0 ACT1 3 3 0x2
4832 0 RD0  3 3 0x1b5
4832 0 RD1  3 3 0x1b5
4928 0 PRE  3 3 

4930 0 ACT0 4 0 0x4
4930 0 ACT1 4 0 0x4
5008 0 WR0  4 0 0x1c4
5008 0 WR1  4 0 0x1c4
5104 0 PRE  4 0 

5106 0 ACT0 5 1 0x8
5106 0 ACT1 5 1 0x8
5184 0 RD0  5 1 0x1d3
5184 0 RD1  5 1 0x1d3
5280 0 PRE  5 1 

5282 0 ACT0 6 2 0x10
5282 0 ACT1 6 2 0x10
5360 0 RD0  6 2 0x1e2
5360 0 RD1  6 2 0x1e2
5456 0 PRE  6 2 

5458 0 ACT0 7 3 0x20
5458 0 ACT1 7 3 0x20
5536 0 WR0  7 3 0x1f1
5536 0 WR1  7 3 0x1f1
5632 0 PRE  7 3 

5634 0 ACT0 0 0 0x0
5634 0 ACT1 0 0 0x0
5712 0 RD0  0 0 0xffff0
5712 0 RD1  0 0 0xffff0
5808 0 PRE  0 0 

5888 0 ACT0 0 0 0x0
5888 0 ACT1 0 0 0x0
5966 0 RD0  0 0 0x00
5966 0 RD1  0 0 0x00
6062 0 PRE  0 0 

6064 0 ACT0 1 1 0x1
6064 0 ACT1 1 1 0x1
6142 0 WR0  1 1 0xffff1
6142 0 WR1  1 1 0xffff1
6238 0 PRE  1 1 

