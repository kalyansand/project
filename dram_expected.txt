         2 0 ACT0 0 0 0x0
         2 0 ACT1 0 0 0x0
         2 0  RD0 0 0 0x0
         2 0  RD1 0 0 0x0
         2 0  PRE 0 0
         4 0 ACT0 1 1 0x1
         4 0 ACT1 1 1 0x1
         4 0  WR0 1 1 0x3FF
         4 0  WR1 1 1 0x3FF
         4 0  PRE 1 1
         6 0 ACT0 2 2 0x2
         6 0 ACT1 2 2 0x2
         6 0  RD0 2 2 0x1E
         6 0  RD1 2 2 0x1E
         6 0  PRE 2 2
         8 0 ACT0 3 3 0x3
         8 0 ACT1 3 3 0x3
         8 0  RD0 3 3 0x2D
         8 0  RD1 3 3 0x2D
         8 0  PRE 3 3
        10 0 ACT0 4 0 0x4
        10 0 ACT1 4 0 0x4
        10 0  WR0 4 0 0x4C
        10 0  WR1 4 0 0x4C
        10 0  PRE 4 0
        12 0 ACT0 5 1 0x5
        12 0 ACT1 5 1 0x5
        12 0  RD0 5 1 0x8B
        12 0  RD1 5 1 0x8B
        12 0  PRE 5 1
        14 0 ACT0 6 2 0x6
        14 0 ACT1 6 2 0x6
        14 0  RD0 6 2 0x10A
        14 0  RD1 6 2 0x10A
        14 0  PRE 6 2
        16 0 ACT0 7 3 0x7
        16 0 ACT1 7 3 0x7
        16 0  WR0 7 3 0x209
        16 0  WR1 7 3 0x209
        16 0  PRE 7 3
        18 0 ACT0 0 0 0x8
        18 0 ACT1 0 0 0x8
        18 0  RD0 0 0 0x108
        18 0  RD1 0 0 0x108
        18 0  PRE 0 0
        20 0 ACT0 1 1 0x9
        20 0 ACT1 1 1 0x9
        20 0  RD0 1 1 0x87
        20 0  RD1 1 1 0x87
        20 0  PRE 1 1
        22 0 ACT0 2 2 0xA
        22 0 ACT1 2 2 0xA
        22 0  WR0 2 2 0x46
        22 0  WR1 2 2 0x46
        22 0  PRE 2 2
        24 0 ACT0 3 3 0xB
        24 0 ACT1 3 3 0xB
        24 0  RD0 3 3 0x25
        24 0  RD1 3 3 0x25
        24 0  PRE 3 3
        26 0 ACT0 4 0 0xC
        26 0 ACT1 4 0 0xC
        26 0  RD0 4 0 0x14
        26 0  RD1 4 0 0x14
        26 0  PRE 4 0
        28 0 ACT0 5 1 0xD
        28 0 ACT1 5 1 0xD
        28 0  WR0 5 1 0x3
        28 0  WR1 5 1 0x3
        28 0  PRE 5 1
        30 0 ACT0 6 2 0xE
        30 0 ACT1 6 2 0xE
        30 0  RD0 6 2 0x12
        30 0  RD1 6 2 0x12
        30 0  PRE 6 2
        32 0 ACT0 7 3 0xF
        32 0 ACT1 7 3 0xF
        32 0  RD0 7 3 0x21
        32 0  RD1 7 3 0x21
        32 0  PRE 7 3
        34 0 ACT0 0 0 0x10
        34 0 ACT1 0 0 0x10
        34 0  WR0 0 0 0x40
        34 0  WR1 0 0 0x40
        34 0  PRE 0 0
        36 0 ACT0 1 1 0x11
        36 0 ACT1 1 1 0x11
        36 0  RD0 1 1 0x8F
        36 0  RD1 1 1 0x8F
        36 0  PRE 1 1
        38 0 ACT0 2 2 0x12
        38 0 ACT1 2 2 0x12
        38 0  RD0 2 2 0x10E
        38 0  RD1 2 2 0x10E
        38 0  PRE 2 2
        40 0 ACT0 3 3 0x13
        40 0 ACT1 3 3 0x13
        40 0  WR0 3 3 0x20D
        40 0  WR1 3 3 0x20D
        40 0  PRE 3 3
        42 0 ACT0 4 0 0x14
        42 0 ACT1 4 0 0x14
        42 0  RD0 4 0 0x10C
        42 0  RD1 4 0 0x10C
        42 0  PRE 4 0
        44 0 ACT0 5 1 0x15
        44 0 ACT1 5 1 0x15
        44 0  RD0 5 1 0x8B
        44 0  RD1 5 1 0x8B
        44 0  PRE 5 1
        46 0 ACT0 6 2 0x16
        46 0 ACT1 6 2 0x16
        46 0  WR0 6 2 0x4A
        46 0  WR1 6 2 0x4A
        46 0  PRE 6 2
        48 0 ACT0 7 3 0x17
        48 0 ACT1 7 3 0x17
        48 0  RD0 7 3 0x29
        48 0  RD1 7 3 0x29
        48 0  PRE 7 3
        50 0 ACT0 0 0 0x18
        50 0 ACT1 0 0 0x18
        50 0  RD0 0 0 0x18
        50 0  RD1 0 0 0x18
        50 0  PRE 0 0
        52 0 ACT0 1 1 0x19
        52 0 ACT1 1 1 0x19
        52 0  WR0 1 1 0x7
        52 0  WR1 1 1 0x7
        52 0  PRE 1 1
        54 0 ACT0 2 2 0x1A
        54 0 ACT1 2 2 0x1A
        54 0  RD0 2 2 0x16
        54 0  RD1 2 2 0x16
        54 0  PRE 2 2
        56 0 ACT0 3 3 0x1B
        56 0 ACT1 3 3 0x1B
        56 0  RD0 3 3 0x25
        56 0  RD1 3 3 0x25
        56 0  PRE 3 3
        58 0 ACT0 4 0 0x1C
        58 0 ACT1 4 0 0x1C
        58 0  WR0 4 0 0x44
        58 0  WR1 4 0 0x44
        58 0  PRE 4 0
        60 0 ACT0 5 1 0x1D
        60 0 ACT1 5 1 0x1D
        60 0  RD0 5 1 0x83
        60 0  RD1 5 1 0x83
        60 0  PRE 5 1
        62 0 ACT0 6 2 0x1E
        62 0 ACT1 6 2 0x1E
        62 0  RD0 6 2 0x102
        62 0  RD1 6 2 0x102
        62 0  PRE 6 2
        64 0 ACT0 7 3 0x1F
        64 0 ACT1 7 3 0x1F
        64 0  WR0 7 3 0x201
        64 0  WR1 7 3 0x201
        64 0  PRE 7 3
        66 0 ACT0 0 0 0xFFFF
        66 0 ACT1 0 0 0xFFFF
        66 0  RD0 0 0 0x0
        66 0  RD1 0 0 0x0
        66 0  PRE 0 0
      1002 0 ACT0 0 0 0x0
      1002 0 ACT1 0 0 0x0
      1002 0  RD0 0 0 0x0
      1002 0  RD1 0 0 0x0
      1002 0  PRE 0 0
      1004 0 ACT0 1 1 0xFFFF
      1004 0 ACT1 1 1 0xFFFF
      1004 0  WR0 1 1 0x11
      1004 0  WR1 1 1 0x11
      1004 0  PRE 1 1
